northstar/src/runtime/island/mod.rs

```diff
@@ -477,53 +496,65 @@ fn seccomp_filter(container: &Container) -> Option<seccomp::AllowList> {
         .map(|seccomp| seccomp::seccomp_filter(seccomp.iter()))
 }
 
-#[derive(Debug, Serialize, Deserialize, PartialEq)]
-enum Start {
-    // Signal the child to go
-    Start,
-    // Signal the parent that go is received
-    Started,
-    // Signal the child that the parent has terminated
-    Died,
+/// Block all signals of this process and currenty thread
+fn signals_block() {
```

Comment:
Blocking those signals until we can process them looks like a cleaner solution than the one we had before :+1: